src/main/java/com/leinardi/pycharm/mypy/mpapi/MypyRunner.java

```diff
@@ -88,8 +94,20 @@ public static boolean isMypyPathValid(String mypyPath, Project project) {
         try {
             process = cmd.createProcess();
             process.waitFor();
+            String error = new BufferedReader(new InputStreamReader(process.getErrorStream()))
+                    .lines().collect(Collectors.joining("\n"));
+            if (!StringUtil.isEmpty(error)) {
+                LOG.error("Error while checking Mypy path: " + error);
+            }
+            String output = new BufferedReader(new InputStreamReader(process.getInputStream()))
```

Comment:
**Reporter**: Error Prone
**Rule**: DefaultCharset
**Severity**: WARN
**File**: home/travis/build/leinardi/mypy-pycharm/src/main/java/com/leinardi/pycharm/mypy/mpapi/MypyRunner.java L102

Implicit use of the platform default charset, which can result in differing behaviour between JVM executions or incorrect behavior if the encoding of the data source doesn&#39;t match expectations.&#10;&#10;String output &#61; new BufferedReader(new InputStreamReader(process.getInputStream()))^(see http://errorprone.info/bugpattern/DefaultCharset)Did you mean &#39;String output &#61; new BufferedReader(new InputStreamReader(process.getInputStream(), UTF_8))&#39; or &#39;String output &#61; new BufferedReader(new InputStreamReader(process.getInputStream(), Charset.defaultCharset()))&#39;?


*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a496237746>*